hackathon/templates/hackathon/includes/hackathon_card.html

```diff
@@ -1,11 +1,48 @@
 <article class="card shadow">
-	<div class="card-body">
-		<h5 class="p-orange card-title">{{ hackathon.display_name }}</h5>
-		<h6 class="card-subtitle mb-2 text-muted">{{ hackathon.start_date }} - {{ hackathon.end_date }}</h6>
-		<p class="card-text">{{ hackathon.description }}</p>
-        {% if hackathon.organiser %}
-		<p>Organiser: <a href="#" class="p-blue card-link">{{ hackathon.organiser }}</a></p>
+  <div class="card-body">
+    <h5 class="p-orange card-title">{{ hackathon.display_name }}</h5>
+    <h6 class="card-subtitle mb-2 text-muted">{{ hackathon.start_date }} - {{ hackathon.end_date }}</h6>
+    <p class="card-text">{{ hackathon.description }}</p>
+    {% if hackathon.start_date < now %}
+      Less than
+    {% endif %}
+    {% if hackathon.organiser %}
+      <p>Organiser: <a href="#" class="p-blue card-link">{{ hackathon.organiser }}</a></p>
+    {% endif %}
+    <a href="#" class="btn btn-ci mr-3">Read More</a>
+
+    <!-- Delete button, only visible for admin users -->
+    {% if user.is_authenticated and user.is_superuser %}
+      <button class="btn btn-ci" type="button" data-toggle="modal" data-target="#delete-modal-{{ hackathon.id }}">
+        Delete Event
+      </button>
+    {% endif %}
+  </div>
+</article>
+
+<!-- Modal - pass in hackathon id to modal id to enable rendering details and deleting correct hackathon event -->
+<div class="modal fade" id="delete-modal-{{ hackathon.id }}" tabindex="-1" aria-labelledby="delete-modal"
+     aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title" id="exampleModalLabel">Delete Hackathon</h4>
```

Comment:
Please change the id to something more meaningful or remove it if not needed.

Reply:
This was the default id from the Bootstrap modal template, I missed it. I've removed it now and the change will be visible when I push it to the existing PR 👍